ui/lib/src/socket.ts

```diff
@@ -81,7 +81,7 @@ export function wsAverageLag(): number {
   return siteSocket?.averageLag ?? 0;
 }
 
-const isOnline = () => !('onLine' in navigator) || navigator.onLine;
+const isOnline = () => !('onLine' in navigator) || navigator.onLine || ['localhost', '127.0.0.1'].includes(location.hostname);
```

Comment:
did you include this by accident? i'd rather be able to debug websockets than be nag-free when lacking internet.

Reply:
Ah I see, I'll revert that then.